src/manifest.json

```diff
@@ -29,8 +29,7 @@
       "matches": ["*://*/*"],
       "run_at": "document_start",
```

Comment:
Would be also need "all_frames": true? Do we see trackers that come in their own iframe?

Reply:
make sense